packages/modules/viewer-react/src/components/app-ui/providers/ViewerStatusbarItemsProvider.tsx

```diff
@@ -0,0 +1,133 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import type {
+  CommonStatusBarItem,
+  UiItemsProvider,
+} from "@itwin/appui-abstract";
+import { StatusBarSection } from "@itwin/appui-abstract";
+import { FooterSeparator } from "@itwin/appui-layout-react";
+import type { DefaultStatusbarItems, StatusBarItem } from "@itwin/appui-react";
+import {
+  FooterModeField,
+  MessageCenterField,
+  SelectionInfoField,
+  SelectionScopeField,
+  SnapModeField,
+  StatusBarItemUtilities,
+  TileLoadingIndicator,
+  ToolAssistanceField,
+  withMessageCenterFieldProps,
+  withStatusFieldProps,
+} from "@itwin/appui-react";
+import * as React from "react";
+
+export class ViewerStatusbarItemsProvider implements UiItemsProvider {
+  public readonly id = "ViewerDefaultStatusbar";
+  private _footerModeOnlySeparator: () => React.ReactNode;
+
+  constructor(private _defaultItems?: DefaultStatusbarItems) {
+    const FooterOnlyDisplay = withStatusFieldProps(FooterModeField);
+    this._footerModeOnlySeparator = (): React.ReactNode => {
+      return (
+        <FooterOnlyDisplay>
+          <FooterSeparator />
+        </FooterOnlyDisplay>
+      );
+    };
+  }
+
+  public provideStatusBarItems(): CommonStatusBarItem[] {
+    const ToolAssistance = withStatusFieldProps(ToolAssistanceField);
+    const MessageCenter = withMessageCenterFieldProps(MessageCenterField);
+    const TileLoadIndicator = withStatusFieldProps(TileLoadingIndicator);
+    const SelectionScope = withStatusFieldProps(SelectionScopeField);
+    const SnapMode = withStatusFieldProps(SnapModeField);
+    const SelectionInfo = withStatusFieldProps(SelectionInfoField);
+    const items: StatusBarItem[] = [];
+
+    if (!this._defaultItems || this._defaultItems.messageCenter) {
```

Comment:
could probably have on if statement wrapping all the others to check for _defaultItems

Reply:
Since its an `||` and not `&&`, I think that would change the behaviour.

Reply:
based on our conversation this morning, can we just remove the message center?

Reply:
I think that was about the `Activity Center`, not the `Message Center`. I already removed the `Activity Center` so you won't see it in this PR